book/src/user/lightwalletd.md

```diff
@@ -0,0 +1,248 @@
+
+```
+[state]
+cache_dir = "/lot/of/free/space/for/zebra"
+...
+```
+
+### Network section
+[#network-section]: #network-section
+
+Only change this if you want to sync for the testnet:
+
+```
+[network]
+...
+network = 'Testnet'
+...
+```
+
+## Sync Zebra
+[#sync-zebra]: #sync-zebra
+
+With the configuration in place you can start synchronizing Zebra with the Zcash blockchain. This may take a while depending on your hardware.
+
+```console
+./target/release/zebrad -c zebrad.toml start
+```
+
+Zebra will display information about sync process:
+
+```console
+...
+zebrad::commands::start: estimated progress to chain tip sync_percent=10.783 % 
+...
+```
+
+Until eventually it will get there:
+
+```console
+...
+zebrad::commands::start: finished initial sync to chain tip, using gossiped blocks sync_percent=100.000 % 
+...
+```
+
+You can interrupt the process at any time with `ctrl-c` and Zebra will resume the next time at around the block you were downloading when stopping the process.
+
+When deploying for production infrastructure, the above command can/should be implemented as a server service or similar configuration. 
+
+For implementing zebra as a service please see [here](https://github.com/ZcashFoundation/zebra/blob/v1.0.0-beta.10/zebrad/systemd/zebrad.service).
+
+## Download and build lightwalletd
+[#download-and-build-lightwalletd]: #download-and-build-lightwalletd
+
+Once you have Zebra running (with open RPC port and synchronized) you can install lightwalletd. There are several versions of lightwalletd in different repositories:
+
+- [The original version](https://github.com/zcash/lightwalletd)
+- [The version used by zecwallet-lite and zecwallet-cli](https://github.com/adityapk00/lightwalletd)
+
+Most of the testing related to Zebra and lightwalletd was done with the zecwallet version in the [adityapk00](https://github.com/adityapk00) repository.
+
+Before installing lightwalletd, you need to have `go` in place.
+
+Please visit [this](https://gist.github.com/nikhita/432436d570b89cab172dcf2894465753) page for go installation/upgrade instructions.
+
+Make sure you have `go` in your path, can add it as:
+
+```console
+echo 'PATH=$PATH:/usr/local/go/bin' >> .profile
+source .profile
+```
+
+With go installed and in your path, download and install lightwalletd:
+
+```console
+git clone https://github.com/adityapk00/lightwalletd
+cd lightwalletd
+make
+make install
+```
+
+If everything went good you should have a `lightwalletd` binary in the root of the project.
+
+## Sync lightwalled
+[#sync-lightwalletd]: (#sync-lightwalletd)
+
+Please make sure you have zebrad running(with RPC endpoint and up to date blockchain) to synchronize lightwalletd.
+
+```console
+./lightwalletd --no-tls-very-insecure --zcash-conf-path ~/.zcash/zcash.conf --data-dir . --log-file /dev/stdout
+```
+
+Lightwalletd expects a `zcash.conf` file with just the following content:
+
+```
+txindex=1
+insightexplorer=1
+experimentalfeatures=1
```

Comment:
These are `zcashd` settings, they are not required for zebrad/lightwalletd.

These settings are the only ones that `lightwalletd` reads, and they are optional.

Can you move this section to an "advanced usage" section at the end?

```suggestion
rpcbind=127.0.0.1
rpcport=8232
```

Reply:
Thanks, done as part of https://github.com/ZcashFoundation/zebra/commit/4e703fc6cbda58f4bb85d10291a22c6707f94b8f